### PR TITLE
Add opendata_mos_household_services_ru spider (~19k locations).

### PR DESCRIPTION
**Pull request by panasonic-mafia**
```
{'atp/category/amenity/car_wash': 686,
 'atp/category/craft/carpenter': 32,
 'atp/category/craft/clockmaker': 239,
 'atp/category/craft/electronics_repair': 1497,
 'atp/category/craft/jeweller': 273,
 'atp/category/craft/key_cutter': 280,
 'atp/category/craft/shoemaker': 467,
 'atp/category/craft/tailor': 1513,
 'atp/category/leisure/sauna': 253,
 'atp/category/shop/beauty': 7328,
 'atp/category/shop/bicycle': 67,
 'atp/category/shop/car_repair': 1874,
 'atp/category/shop/copyshop': 469,
 'atp/category/shop/dry_cleaning': 992,
 'atp/category/shop/frame': 91,
 'atp/category/shop/funeral_directors': 38,
 'atp/category/shop/hairdresser': 1928,
 'atp/category/shop/laundry': 98,
 'atp/category/shop/pawnbroker': 479,
 'atp/category/shop/photo': 666,
 'atp/field/brand/missing': 19270,
 'atp/field/brand_wikidata/missing': 19270,
 'atp/field/city/missing': 19270,
 'atp/field/country/from_spider_name': 19270,
 'atp/field/email/missing': 19270,
 'atp/field/image/missing': 19270,
 'atp/field/opening_hours/missing': 5402,
 'atp/field/phone/invalid': 83,
 'atp/field/phone/missing': 6784,
 'atp/field/postcode/missing': 19270,
 'atp/field/state/missing': 19270,
 'atp/field/street_address/missing': 19270,
 'atp/field/twitter/missing': 19270,
 'atp/field/website/missing': 19270,
 'atp/opendata_mos_ru/category/failed/иные объекты бытового обслуживания': 395,
 'atp/opendata_mos_ru/category/failed/комплексное предприятие бытового обслуживания': 1205,
 'atp/opendata_mos_ru/category/failed/услуги проката': 24,
 'atp/opendata_mos_ru/category/failed/услуги профессиональной уборки - клининговые услуги': 4,
 'atp/opendata_mos_ru/hours/failed': 1549,
 'downloader/request_bytes': 27062,
 'downloader/request_count': 44,
 'downloader/request_method_count/GET': 44,
 'downloader/response_bytes': 39050471,
 'downloader/response_count': 44,
 'downloader/response_status_count/200': 43,
 'downloader/response_status_count/400': 1,
 'elapsed_time_seconds': 27.196609,
 'feedexport/success_count/FileFeedStorage': 1,
 'finish_reason': 'finished',
 'finish_time': datetime.datetime(2023, 7, 1, 11, 31, 0, 48289),
 'item_scraped_count': 19270,
 'log_count/INFO': 10,
 'memusage/max': 116523008,
 'memusage/startup': 116523008,
 'request_depth_max': 1,
 'response_received_count': 44,
 'robotstxt/request_count': 1,
 'robotstxt/response_count': 1,
 'robotstxt/response_status_count/400': 1,
 'scheduler/dequeued': 43,
 'scheduler/dequeued/memory': 43,
 'scheduler/enqueued': 43,
 'scheduler/enqueued/memory': 43,
 'start_time': datetime.datetime(2023, 7, 1, 11, 30, 32, 851680)}
```

### CODE REVIEW

**Comment by panasonic-mafia** [comment]
Looks like this spider requires RU or NL ip in order to work.

File: locations/categories.py
Comment (davidhicks) [inline locations/categories.py:29]: Extra T by accident in CRAFTT

File: locations/spiders/opendata_mos_household_services_ru.py
Comment (davidhicks) [inline locations/spiders/opendata_mos_household_services_ru.py:155]: Do you have an example of the opening hours formats that failed? Perhaps `OpeningHours.add_ranges_from_string(..)` could help with parsing?

**Review by davidhicks — CHANGES_REQUESTED** [review]

File: locations/categories.py
Comment (panasonic-mafia) [reply locations/categories.py]: Thanks! Fixed.

File: locations/spiders/opendata_mos_household_services_ru.py
Comment (panasonic-mafia) [reply locations/spiders/opendata_mos_household_services_ru.py]: Fixed.

**Comment by panasonic-mafia** [comment]
Run stats with fixed opening hours:

```
{'atp/category/amenity/car_wash': 686,
 'atp/category/craft/carpenter': 32,
 'atp/category/craft/clockmaker': 239,
 'atp/category/craft/electronics_repair': 1497,
 'atp/category/craft/jeweller': 273,
 'atp/category/craft/key_cutter': 280,
 'atp/category/craft/shoemaker': 467,
 'atp/category/craft/tailor': 1513,
 'atp/category/leisure/sauna': 253,
 'atp/category/shop/beauty': 7328,
 'atp/category/shop/bicycle': 67,
 'atp/category/shop/car_repair': 1874,
 'atp/category/shop/copyshop': 469,
 'atp/category/shop/dry_cleaning': 992,
 'atp/category/shop/frame': 91,
 'atp/category/shop/funeral_directors': 38,
 'atp/category/shop/hairdresser': 1928,
 'atp/category/shop/laundry': 98,
 'atp/category/shop/pawnbroker': 479,
 'atp/category/shop/photo': 666,
 'atp/field/brand/missing': 19270,
 'atp/field/brand_wikidata/missing': 19270,
 'atp/field/city/missing': 19270,
 'atp/field/country/from_spider_name': 19270,
 'atp/field/email/missing': 19270,
 'atp/field/image/missing': 19270,
 'atp/field/opening_hours/missing': 4005,
 'atp/field/phone/invalid': 83,
 'atp/field/phone/missing': 6784,
 'atp/field/postcode/missing': 19270,
 'atp/field/state/missing': 19270,
 'atp/field/street_address/missing': 19270,
 'atp/field/twitter/missing': 19270,
 'atp/field/website/missing': 19270,
 'atp/opendata_mos_ru/category/failed/иные объекты бытового обслуживания': 395,
 'atp/opendata_mos_ru/category/failed/комплексное предприятие бытового обслуживания': 1205,
 'atp/opendata_mos_ru/category/failed/услуги проката': 24,
 'atp/opendata_mos_ru/category/failed/услуги профессиональной уборки - клининговые услуги': 4,
 'downloader/exception_count': 1,
 'downloader/exception_type_count/twisted.internet.error.TimeoutError': 1,
 'downloader/request_bytes': 27687,
 'downloader/request_count': 45,
 'downloader/request_method_count/GET': 45,
 'downloader/response_bytes': 39050471,
 'downloader/response_count': 44,
 'downloader/response_status_count/200': 43,
 'downloader/response_status_count/400': 1,
 'elapsed_time_seconds': 282.517084,
 'feedexport/success_count/FileFeedStorage': 1,
 'finish_reason': 'finished',
 'finish_time': datetime.datetime(2023, 7, 1, 15, 24, 18, 711443),
 'item_scraped_count': 19270,
 'log_count/INFO': 14,
 'memusage/max': 199979008,
 'memusage/startup': 116301824,
 'request_depth_max': 1,
 'response_received_count': 44,
 'retry/count': 1,
 'retry/reason_count/twisted.internet.error.TimeoutError': 1,
 'robotstxt/request_count': 1,
 'robotstxt/response_count': 1,
 'robotstxt/response_status_count/400': 1,
 'scheduler/dequeued': 44,
 'scheduler/dequeued/memory': 44,
 'scheduler/enqueued': 44,
 'scheduler/enqueued/memory': 44,
 'start_time': datetime.datetime(2023, 7, 1, 15, 19, 36, 194359)}
```

File: locations/spiders/opendata_mos_household_services_ru.py
Comment (iandees) [inline locations/spiders/opendata_mos_household_services_ru.py:159]:
```suggestion
            except Exception:
```

**Comment by panasonic-mafia** [comment]
@iandees would you be able to merge it please? `spider-pr` action is failing as this spider needs RU or NL ip in order to work.

**Comment by iandees** [comment]
I was waiting for a re-review from @davidhicks, but it looks like you covered his concerns.

**Comment by panasonic-mafia** [comment]
Thanks all.

**Pull request merged**
